cupboard_app/views.py

```diff
@@ -784,3 +791,83 @@ def list(self, request: Request) -> Response:
         all_measurements = get_all_measurements()
         serializer = MeasurementSerializer(all_measurements, many=True)
         return Response(serializer.data, status=200)
+        return Response(serializer.data)
+
+
+@extend_schema(tags=['CustomIngredients'])
+class CustomIngredientsViewSet(viewsets.ViewSet):
+    MISSING_ING = 'Missing ingredient and type in message body.'
+
+    @extend_schema(
+        request=None,
+        responses={
+            201: CustomIngredientSerializer,
+            400: MessageSerializer,
+            401: auth_failed_response,
+        },
+        examples=[
+            OpenApiExample(
```

Comment:
Add a OpenApiExample for the request body. And specify this is for the request with `request_only=True`

```
OpenApiExample(
                name='Create Custom Ingredient',
                value={
                    'ingredient': 'Homemade Meatball',
                    'type': 'custom
                },
                request_only=True
            ),
```